module/VuFind/src/VuFind/Log/LoggerFactory.php

```diff
@@ -367,12 +368,11 @@ protected function addHandlers(
             // Ensure verbosity is an int, default to 1 if not specified or invalid
             $verbosity = isset($parts[1]) && is_numeric($parts[1]) ? (int)$parts[1] : 1;
 
-            $min = LogLevel::DEBUG; // Default min, will be overwritten by switch
-            $max = LogLevel::EMERGENCY; // Default max, will be overwritten by switch
```

Comment:
Why did you remove these? It appears that without them, there is a possibility that a FilterHandler will be passed uninitialized values in line 421 when an unexpected level value is passed in the configuration (unless I'm misreading something).

Maybe we need a test case that passes a gibberish level to see what happens (and capture our expectations in a test).

Reply:
Done, apart from logging the bad logging level. But now it'd be possible!